rex/multi_time_resource.py

```diff
@@ -299,12 +306,19 @@ def _map_file_instances(file_paths, res_cls=Resource, **res_cls_kwargs):
 
         Returns
         -------
-        h5_map : dict
-            Dictionary mapping file paths to open resource instances
+        h5_map : pd.DataFrame
+            DataFrame mapping file paths to open resource instances and
+            datasets per file (columns: fp, h5, and dsets)
         """
-        h5_map = {}
-        for f_path in file_paths:
-            h5_map[f_path] = res_cls(f_path, **res_cls_kwargs)
+        h5_map = pd.DataFrame({'fp': file_paths, 'h5': None,
+                               't0': None, 't1': None})
+        for i, f_path in enumerate(file_paths):
+            h5_map.at[i, 'h5'] = res_cls(f_path, **res_cls_kwargs)
+            h5_map.at[i, 't0'] = h5_map.at[i, 'h5'].time_index.values[0]
+            h5_map.at[i, 't1'] = h5_map.at[i, 'h5'].time_index.values[1]
```

Comment:
What is the point of `t1` here? I can't seem to find it used anywhere else in the code

Reply:
it's really information/debugging purposes haha. This h5map is a great way to store meta data in-situ and the t0/t1 are both useful in this regard.